Dockerfile

```diff
@@ -54,9 +53,4 @@ RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key
   && apt-get install -y google-chrome-unstable --no-install-recommends \
   && rm -rf /var/lib/apt/lists/*
 
-ADD . ./
-
-ARG is_testing
```

Comment:
Remove this logic since it's not longer necessary, adding testing requirements is handled explicitly in CI.